staging/src/k8s.io/kubectl/pkg/cmd/cmd.go

```diff
@@ -96,15 +96,15 @@ type KubectlOptions struct {
 	genericiooptions.IOStreams
 }
 
-var defaultConfigFlags = genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag().WithDiscoveryBurst(300).WithDiscoveryQPS(50.0)
```

Comment:
rather than repeating this in two places, make this a private function we can call if needed

```go
func defaultConfigFlags() *genericclioptions.ConfigFlags {
 
 return genericclioptions.NewConfigFlags(true).WithDeprecatedPasswordFlag().WithDiscoveryBurst(300).WithDiscoveryQPS(50.0)
}
```